gapic-generator/lib/gapic/schema/api.rb

```diff
@@ -136,13 +136,7 @@ def protoc_options
       # Reconstructed string representation of the protoc options
       # @return [String]
       def protoc_parameter
-        protoc_options.map do |k, v|
-          v = Array(v).map do |s|
-            s.gsub("\\", "\\\\\\\\").gsub(",", "\\\\,").gsub("=", "\\\\=")
-          end.join("=")
-          k = key_to_str k
-          "#{k}=#{v}"
-        end.join ","
+        @protoc_parameters.map(&:input_str).join(",")
```

Comment:
Also move this reconstruct-string-representation logic into the parser class.

Reply:
fixed